package.json

```diff
@@ -1,13 +1,21 @@
 {
   "name": "linter-clojure",
-  "linter-package": true,
   "main": "./lib/init",
-  "version": "0.0.5",
+  "version": "0.0.6",
   "description": "Lint Clojure on the fly, using clojure-x.x.x.jar",
   "repository": "https://github.com/AtomLinter/linter-clojure",
   "license": "MIT",
   "engines": {
-    "atom": ">0.50.0"
+    "atom": ">=1.0.0 <2.0.0"
   },
-  "dependencies": {}
+  "providedServices": {
+    "linter": {
+      "versions": {
+        "1.0.0": "provideLinter"
+      }
+    }
+  },
+  "dependencies": {
+    "atom-linter": "3.2.0"
```

Comment:
I can update this if you like. Usually, for libraries like this, I just pin the package to the exact version I used to write it with. That way I don't have to keep testing with newer versions that might break since not everyone follows strict semver rules.